support/http/logging_middleware_test.go

```diff
@@ -29,18 +30,35 @@ func TestHTTPMiddleware(t *testing.T) {
 	// get the log buffer and ensure it has both the start and end log lines for
 	// each request
 	logged := done()
-	if assert.Len(t, logged, 4, "unexpected log line count") {
+	if assert.Len(t, logged, 5, "unexpected log line count") {
 		assert.Equal(t, "starting request", logged[0].Message)
-		assert.Equal(t, "starting request", logged[2].Message)
-		assert.Equal(t, "finished request", logged[1].Message)
-		assert.Equal(t, "finished request", logged[3].Message)
-	}
+		assert.Equal(t, "http", logged[0].Data["subsys"])
+		assert.Equal(t, "GET", logged[0].Data["method"])
+		assert.NotEmpty(t, logged[0].Data["req"])
+		assert.NotEmpty(t, logged[0].Data["path"])
+		req1 := logged[0].Data["req"]
+
+		assert.Equal(t, "handler log line", logged[1].Message)
+		assert.Equal(t, req1, logged[0].Data["req"])
```

Comment:
I don't see the purpose of testing `assert.Equal(t, req1, logged[0].Data["req"])` if you just defined it in line `39`. Should this be `logged[1]`, instead?

```suggestion
		assert.Equal(t, req1, logged[1].Data["req"])
```